data.proto

```diff
@@ -126,3 +121,26 @@ message RunConfig {
   uint32 framestack = 27;
   ReplayBufferConfig replay_buffer_config = 28;
 }
+
+message ModelArchitectureConfig {
+  uint32 hidden_size = 1;
+}
+
+message SimpleA2CTrainingConfig {
+  uint32 epoch_count = 1;
+  uint32 epoch_trial_count = 2;
```

Comment:
`trial_count` ?

Reply:
It's the number of trial per epoch so it made sense to use `epoch_trial_count` I can change it if you think it makes more sense